qatrfm/environment.py

```diff
@@ -27,15 +27,16 @@
 from qatrfm.domain import Domain
 from qatrfm.utils.logger import QaTrfmLogger
 from qatrfm.utils import libutils
+from pathlib import Path
 
 
 class TerraformEnv(object):
 
     logger = QaTrfmLogger.getQatrfmLogger(__name__)
-    BASEDIR = '/root/terraform/'
 
     def __init__(self, image, tf_file=None, num_domains=1,
-                 cores=1, ram=1024, snapshots=False):
+                 cores=1, ram=1024, snapshots=False,
+                 base_working_dir='./qatrfm_working'):
```

Comment:
```suggestion
                 base_dir='./qatrfm_workdir'):
```

`base_working_dir` is similar to later `self.workdir`, maybe just `base_dir` is enough?